src/tests/ut/receptionist/shift_test.cpp

```diff
@@ -1,6 +1,7 @@
 #include "shift_test.hpp"
+#include "shift.hpp"
 
-namespace
+namespace unit_test::receptionist
```

Comment:
Please revert it to anonymous namespace 

[cpp coreguidlines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnon-exported-entities)

If we want to include namespaces we shall speak about it's structure and start from source code

Reply:
Done